### ISSUE TITLE
revisionable creates duplicate revisions in the database

### ISSUE THREAD

**Issue by comdexxsolutionsllc** [VentureCraft/revisionable#284]
revisionable creates duplicate revisions in the database

How do I fix this?

**Comment by astierler** [comment]
@comdexxsolutionsllc  can you post a basic example of code where this is happening?

Could you possibly have some logic that is processing a save twice?  This may be happening without you realizing, but only just now noticed because of duplicate revision entries.

I've never ran into this using it across various projects, so curious what could cause this

**Comment by acacha** [comment]
Same problem here and still looking for a solution...

**Comment by acacha** [comment]
Ok in my case it seem I have added to my model booth the Trait RevisionableTrait and extends Revisionable:

```php
class Job extends Revisionable
{
    use RevisionableTrait;
...
```

**Comment by acacha** [comment]
ping @comdexxsolutionsllc @astierler

**Comment by astierler** [comment]
@acacha yes, that'd do it.  The "Revisionable" Model is there for legacy Laravel support where the traits wouldn't work.  @comdexxsolutionsllc if you did the same thing, definitely try just using the trait.

Thanks for pointing out a possible solution @acacha
